Add max import error (#488)
directus/docs: content/guides/02.content/4.import-export.md

permissions for the related collection.
::

::callout{icon="material-symbols:info-outline"}
**Import Error Handling**

> During import operations, validation errors (field validation, database constraints, etc.) are collected up to a maximum limit. Once the `MAX_IMPORT_ERRORS` environment variable limit is reached, the import stops and returns all collected errors.

Reviewer: ```suggestion
During import operations, errors are collected up to the maximum defined by [`MAX_IMPORT_ERRORS`](content/configuration/security-limits.md), after which the import is cancelled.
```
